1-mandatory/3-slideshow/index.html

```diff
@@ -11,7 +11,36 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <section area-label = "Newest Photos">
```

Comment:
Good work I like this. I think it wants a second more complex iteration as well but for what it is this is very nice